.github/pull_request_template.md

```diff
@@ -9,12 +9,15 @@
 1. `x.java`
 2. `y.python`
 
+## 🚨 User Impact 🚨
+Are there any breaking changes? 
+What is the end result perceived by the user?
+
 ## Pre-merge Checklist
 Expand the relevant checklist and delete the others. 
 
 <details><summary> <strong> New Connector </strong></summary>
 <p>
-   
```

Comment:
Oh, sorry that I did not mean to delete this whole line. Just wanted to remove the newly-introduced spaces.